controllers/v1/submissionsController.js

```diff
@@ -1205,7 +1185,8 @@ module.exports = class Submission extends Abstract {
             let updateObject = {}
 
             updateObject.$set = {
-              criterias: criteriaData
+              criterias: criteriaData,
+              ratingCompletedAt : new Date()
```

Comment:
I don't think the ratingCompletedAt key is there in submission model.

@aman-tunerlabs